### PR TITLE
Use hypersistence for json serialization

### PR DESCRIPTION
**Pull request by kbirk**

### CODE REVIEW

File: packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/Grounding.java [hunk 1]
```diff
@@ -23,12 +26,14 @@ public class Grounding implements Serializable {
 	private static final long serialVersionUID = 302308407252037615L;
 
 	/** Ontological identifier per DKG */
-	@JdbcTypeCode(SqlTypes.JSON)
+	@Type(JsonType.class)
+	@Column(columnDefinition = "text")
```
Comment (dvince2) [inline packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/Grounding.java:30]: These will require a migration

Comment (dvince2) [reply packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/Grounding.java:30]: (and the other type changes will too - I won't comment spam on them :)

**Review by dvince2 — APPROVED** [review]
I think this looks good, but requires a migration for the change in field types.

File: packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/Grounding.java (hunk 1 above)
Comment (kbirk) [reply packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/Grounding.java]: No migration needed, as we reverted datasets and these Groundings will all be created fresh in their own table.

**Pull request merged**